### ISSUE TITLE
Lightning UrlVerification fails signature validation

### ISSUE THREAD

**Issue by jmccance** [seratch/jslack#254]
When using Lightning, the UrlVerification request gets rejected because of a bad signature.

```
{
  "severity": "INFO",
  "logger": "com.github.seratch.jslack.lightning.middleware.builtin.RequestVerification",
  "message": "Invalid signature detected - v0=6c3abdbe7c4d8dc1132d1c7da746e25f5663d077ebb857fcaa8df6cd66a4e379"
}
```

Tracing through the code, the issue seems to be that — unlike other Requests — the UrlVerificationRequest doesn't return the full request body when `getRequestBodyAsString` is called. It only returns the challenge. Since calculating the signature requires the whole request body, the signature validation fails, the endpoint returns 401, and Slack won't validate the event callback URL.

**Comment by jmccance** [comment]
Tomorrow I'll take a look at what's entailed in submitting a PR to fix. Seems relatively straightforward. 🙂

**Comment by seratch** [comment]
This issue has been fixed since version 3.0.3.
